src/pages/index.js

```diff
@@ -398,7 +406,11 @@ const Index = ({ data: initialData }) => {
           ) : null}
           <div className={classes.fullContainer}>
             <div className={classes.topContainer}>
+              <Link className={classes.feedbackLink} to={surveyLink}>
+                <Typography>FEEDBACK</Typography>
+              </Link>
               <div className={classes.userMenuContainer}>
+                <InviteFriendsIconContainer user={user} />
```

Comment:
A few styling changes:
* Let's add a little more horizontal spacing around the icon so it matches the distance between other items in the menu
* This icon is bigger than the settings icon. Try 20x20px to match?
<img width="585" alt="Screen Shot 2021-03-22 at 2 09 43 PM" src="https://user-images.githubusercontent.com/5924325/112058830-735c0180-8b31-11eb-8f9b-903ca0918d28.png">

Reply:
20x20 looked kind of off, the icon just seemed smaller than the settings but I can push and get your eyes on it

Reply:
I ended up going 24x24, which is the size of the paw icon.  The social share icon is weird tho as the svg doesn't use up all of the 24px vertical height, which is why going 20x20 looked weird.  LMK your thoughts.  Fixed the spacing

Reply:
Style feedback:
* I agree with you about the icon feeling small. Maybe split the difference and go 28px?
* The icon button is now elliptical but should be circular (see pic below). Make sure the margin is on the top-level element.
* Add some top margin on the popover so its top is in line with the other popovers in the menu
* Maybe `gutterBottom` on the popover text or some top/bottom margin around the invite friends component? Feels cramped.
<img width="1437" alt="Screen Shot 2021-03-22 at 8 02 21 PM" src="https://user-images.githubusercontent.com/5924325/112087251-fbf39580-8b63-11eb-95b9-5f33df39ba0b.png">

Reply:
pushed some style updates. lmk what you think

Reply:
This looks great to me!